library/src-bootstrapped/scala/quoted/Type.scala

```diff
@@ -0,0 +1,46 @@
+package scala.quoted
+
+import scala.quoted.Types.TaggedType
+import scala.reflect.ClassTag
+import scala.runtime.quoted.Unpickler.Pickled
+
+sealed abstract class Type[T <: AnyKind] {
```

Comment:
Here I only added ` <: AnyKind` and `src-non-bootstrapped/scala/quoted/Type.scala` contatins the current version without the bound.